model/vm_host.rb

```diff
@@ -85,8 +86,7 @@ def ip6_random_vm_network(prefix = 79)
     # The upper range is updated later to increase the entropy if the host has
```

Comment:
Are these descriptions still legit? Seems like the algorithm/design theory has changed quite a bit?

Reply:
Updated them